coderd/users.go

```diff
@@ -192,9 +193,12 @@ func (api *API) postFirstUser(rw http.ResponseWriter, r *http.Request) {
 			Username:        createUser.Username,
 			Name:            createUser.Name,
 			Password:        createUser.Password,
+			UserStatus:      ptr.Ref(codersdk.UserStatusActive),
 			OrganizationIDs: []uuid.UUID{defaultOrg.ID},
 		},
-		LoginType:          database.LoginTypePassword,
+		LoginType: database.LoginTypePassword,
+		// There's no reason to create the first user as dormant, since you have
```

Comment:
shouldn't this comment go above the user status field?

Reply:
Ah yeah, I moved the field, forgot to move the comment too.